webui/src/components/editrecord.jsx

```diff
@@ -616,23 +712,44 @@ const EditRecord = React.createClass({
     },
 
     renderSubmitDraftForm() {
-        const klass = this.state.waitingForServer ? 'disabled' :
-                      this.isForPublication() ? 'btn-primary btn-danger' :
-                      this.state.dirty ? 'btn-primary' : 'disabled';
-        const text = this.state.waitingForServer ? "Updating record, please wait..." :
-                      this.isForPublication() ? 'Save and Publish' :
-                      this.state.dirty ? 'Save Draft' : 'The draft is up to date';
-        return (
-            <div className="col-sm-offset-3 col-sm-9">
-                <label style={{fontSize:18, fontWeight:'normal'}}>
-                    <input type="checkbox" value={this.isForPublication} onChange={this.setPublishedState}/>
-                    {" "}Submit draft for publication
-                </label>
-                <p>When the draft is published it will be assigned a PID, making it publicly citable.
-                    But a published record's files can no longer be modified by its owner. </p>
-                <button type="submit" className={"btn btn-default btn-block "+klass} onClick={this.updateRecord}>{text}</button>
-            </div>
-        );
+        if(this.props.community){
```

Comment:
I'd prefer an error here: if the community is not defined, the user cannot do anything anyway (the buttons are not rendered). This should not happen 😄

Reply:
ok, will fix it